PluralKit.Core/Database/Migrations/ModelRepository.Reminder.cs

```diff
@@ -0,0 +1,23 @@
+using Dapper;
+using NodaTime;
+using System.Threading.Tasks;
+
+namespace PluralKit.Core {
+    //TODO: Method to mark reminders as unseen
+    public partial class ModelRepository {
+        public async Task AddReminder(IPKConnection conn, PKReminder reminder) {
+            await conn.ExecuteAsync("insert into reminders(mid, channel, guild, member, system, seen) values (@Mid, @Channel, @Guild, @Member, @System, @Seen)", reminder);
+        }
+
+    }
+
+    public class PKReminder {
+        public ulong Mid { get; set; }
+        public ulong Channel { get; set; }
+        public ulong? Guild { get; set; }
+        public MemberId? Member { get; set; }
```

Comment:
When is MemberId null?

Reply:
someone asked for system wide reminders that aren't targeted towards a specific member - that hasn't been implemented yet, but it would be for that use case.

Reply:
Ah :+1: